src/middleware/github-server-app-middleware.ts

```diff
@@ -1,31 +1,54 @@
 import { NextFunction, Request, Response } from "express";
 import { GitHubServerApp } from "models/github-server-app";
 import { Installation } from "models/installation";
+import { validationResult } from "express-validator";
+import { envVars } from "config/env";
 
 export const GithubServerAppMiddleware = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+
 	const { jiraHost } = res.locals;
-	const { id } = req.params;
+	const { uuid } = req.params;
+
+	req.log = req.log.child({ uuid, jiraHost });
 
-	if (id) {
-		req.log.debug(`Retrieving GitHub app with id ${id}`);
-		const gitHubServerApp = await GitHubServerApp.getForGitHubServerAppId(Number(id));
+	if (uuid && validationResult(req).isEmpty()) {
```

Comment:
Could `validationResult(req).isEmpty()` include other validation errors from up the chain?

Reply:
Good call! 
I suspect we can simply remove the `validationResult` here since this middleware doesn't specific any validation rules, so nor should it validate the result. 
Maybe the upper router should do the check.
Let me come back to you.

Reply:
@joshkay10 , I've removed the `validationResult.isEmtpy()` here because:
1. There's no `param("uuid").isXXX()` settings in this middleware, therefore I don't think we need any validation in this middleware. Not its job.
2. In fact the validation is set one level up in the router via regex. Express will direct the traffic to 404 if uuid not valid. I think it make sense in this case, so nothing else we need to do.
3. I've added one test to test the uuid invalid in upper level router. 

Please let me know what you thinks. Thanks :)

Reply:
Yeah nice, I agree, bit tidier.

3) test. nice!